library/std/src/prelude/mod.rs

```diff
@@ -1,4 +1,4 @@
-//! # The Rust Prelude
+//! The Rust Prelude.
```

Comment:
I made this change to be consistent with the headings for `std` and `core` ("The Rust Standard Library" and "The Rust Core Library").

> Both how the line “The Rust Prelude” is more indented than every other headline here

That's something rustdoc does; we could consider changing that. Also, this is true for the `std` and `core` headlines, I don't think it's any different from here.

> (it says “prelude” twice)

It says "prelude" twice regardless of whether you use a heading though, right?

-----

But, I don't feel super strongly about this either way. I think I still prefer using a heading, but either way seems OK enough.

Reply:
> It says "prelude" twice regardless of whether you use a heading though, right?

Right, right… Maybe the point can be made that `std`, `alloc`, or `core` aren’t self-descriptive enough, and—at least for `alloc` and `std`—the additional headline gives more information. A module named `prelude` doesn’t _really_ need an additional headline stating that is’s “The Prelude”.

Thinking about this some more, `std::prelude` isn’t actually the prelude. Instead, `std::prelude::v1` is. Maybe this could be an argument in favor of using a headline since it isn’t 100% accurate as a self-description.
> either way seems OK enough if people prefer one over the other

Sure. I’m not feeling too strongly about this either. Now, how do we find out what “people” prefer? I guess, as long as we don’t know, I might change it back to the status quo.